include/fmt/core.h

```diff
@@ -392,11 +392,14 @@ template <typename Char> class basic_string_view {
     the size with ``std::char_traits<Char>::length``.
     \endrst
    */
-#if __cplusplus >= 201703L  // C++17's char_traits::length() is constexpr.
-  FMT_CONSTEXPR
+#if __cplusplus >= 201703L
```

Comment:
Please revert back `  // C++17's char_traits::length() is constexpr.` comment to this line.